lib/src/snapshotter/snapshotter.dart

```diff
@@ -45,10 +45,10 @@ final class Snapshotter {
     this.onStyleDataLoadedListener,
     this.onStyleImageMissingListener,
   }) {
-    final messenger = ProxyBinaryMessenger(suffix: _suffix.toString());
-    _snapshotterMessenger = _SnapshotterMessenger(binaryMessenger: messenger);
-    style = StyleManager(binaryMessenger: messenger);
-    _mapEvents = _MapEvents(binaryMessenger: messenger);
+    _snapshotterMessenger =
+        _SnapshotterMessenger(messageChannelSuffix: _suffix.toString());
```

Comment:
I noticed a pattern where binary messenger is specified explicitly, for consistency it should be specified here as well.

Reply:
I don't think it is necessary, as stated by the doc, so we can leave it as optional null
> The [binaryMessenger] named argument is available for dependency injection.  If it is left null, the default BinaryMessenger will be used which routes to the host platform.